superset-frontend/plugins/plugin-chart-table/src/Styles.tsx

```diff
@@ -27,29 +27,66 @@ export default styled.div`
     margin: 0;
   }
 
-  th,
-  td {
+  tr {
+    width: 656px;
+    height: 32px;
+  }
+
+  #bootstrap_overrides > td, tbody > tr > td, .table > tfoot > tr > td {
     min-width: 4.3em;
+    border: 1px solid #E6E6E7;
+    vertical-align: middle;
+    font-family: Inter, serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 13px;
+    letter-spacing: -0.01em;
+    text-transform: capitalize;
+    font-feature-settings: 'tnum' on, 'lnum' on;
+    color: #535355;
+  }
+
+  tr {
+    border: 1px solid #E6E6E7;
   }
 
   thead > tr > th {
     padding-right: 1.4em;
     position: relative;
-    background: ${({ theme: { colors } }) => colors.grayscale.light5};
+    background: ${({theme: {colors}}) => colors.grayscale.light5};
     text-align: left;
+    padding-left: 15px;
+    width: 178px;
+    height: 60px;
+    display: table-cell;
+    vertical-align: middle;
+    border: 1px solid #E6E6E7;
+    font-family: Inter, serif;
+    font-style: normal;
+    font-weight: 550;
+    font-size: 13px;
+    line-height: 12px;
+    letter-spacing: 0.02em;
+    text-transform: capitalize;
+    font-feature-settings: 'tnum' on, 'lnum' on;
+    color: #6C6C6E;
   }
   th svg {
     color: ${({ theme: { colors } }) => colors.grayscale.light2};
     position: relative;
     vertical-align: middle;
-    margin: 0 ${({ theme: { gridUnit } }) => gridUnit}px;
+    margin: 0 ${({theme: {gridUnit}}) => gridUnit}px;
```

Comment:
same question, was this run through prettier?

Reply:
It wasn't i think IDE edited it. I'll revert it back to the way it was before.